Update pod-lifecycle.md (#22939)
kubernetes/website: content/en/docs/concepts/workloads/pods/pod-lifecycle.md

### `Terminated` {#container-state-terminated}

> A container in the `Terminated` state had began execution and then has either run to
> completion or has failed for some reason. When you use `kubectl` to query a Pod with
> a container that is `Terminated`, you see a reason, and exit code, and the start and
> finish time for that container's period of execution.

Reviewer: ```suggestion
A container in the `Terminated` state began execution and then either ran to
completion or failed for some reason. When you use `kubectl` to query a Pod with
a container that is `Terminated`, you see a reason, an exit code, and the start and
finish time for that container's period of execution.
```
